src/plots/cartesian/axes.js

```diff
@@ -249,7 +249,16 @@ axes.handleTickDefaults = function(containerIn, containerOut, coerce, axType, op
 
             if(!tickFormat && axType !== 'date') {
                 coerce('showexponent', showAttrDflt);
-                coerce('exponentformat');
+                coerce('exponentbase');
+
+                var exponentbase = containerIn.exponentbase;
+                var exponentformat = containerIn.exponentformat;
```

Comment:
@mdtusz :cow2: tip 

The coerce function returns the coerced value. So you write:

```js
var exponentBase = coerce('exponentbase');
```

Moreover, we (i.e me and @alexcjohnson) usually write one `coerce` per attribute and write logic around the dynamic default.

```js
var expFormatDflt = (exponentBase !== 2 && exponentBase !== 10) ? 'power' : undefined;

// or if you don't like ternaries
var exFormatDflt;
if(exponentBase !== 2 && exponentBase !== 10) exFormatDflt = 'power';

coerce('exponentformat', exFormatDflt);
```